File: .ccls-cache/@home@runner@Beecrowd-BeginerParte-2/1061.c
#include <stdio.h>

int main(){

int w1,w2,w,x1,x2,x,y1,y2,y,z1,z2,z;
char a[3];
  char b;
  
  scanf("%s %d" , a , &w1);

  scanf("%d %c %d %c %d",  &x1 , &b , &y1 , &b , &z1);

  scanf("%s %d" , a , &w2);

  scanf("%d %c %d %c %d", &x2 , &b , &y2 , &b , &z2);

  //|Dia
  if(w1==w2)
     w=0;

  else if(w1<w2)
    w=w2-w1;

  //Hora
  if(x1==x2)
    x=0;

  else if(x1<x2)
    x = x2-x1;

  else if(x1>x2){
    x = (24-x1) + x2;
    w-=1;
  }

  //Minuto
  if(y1==y2)
    y=0;

  else if(y1<y2)
    y=y2-y1;

  else if(y1>y2){
    y=60-y1+y2;
    x-=1;
  }

  //Segundo
    if(z1==z2)
    z=0;

  else if(z1<z2)
    z=z2-z1;

  else if(z1>z2){
    z=60-z1+z2;
    y-=1;
  }

printf("%d dia(s)\n" , w);

  printf("%d hora(s)\n" , x);

  printf("%d minuto(s)\n" , y);

  printf("%d segundo(s)\n" , z);
  
  
  return 0;
}

//Esta dando erra porém todos os resultados testados estão batendo

File: 1061.py
d1 = input()
t1 = input()
d2 = input()
t2 = input()

d1 = d1[4:]
d1 = int(d1)
d2 = d2[4:]
d2 = int(d2)

h1 = int(t1[:2])
m1 = int(t1[5:7])
s1 = int(t1[10:])

h2 = int(t2[:2])
m2 = int(t2[5:7])
s2 = int(t2[10:])


x1 = s1 + m1*60 + h1*3600 + d1*3600*24
x2 = s2 + m2*60 + h2*3600 + d2*3600*24

x = x2 - x1

d = x//(3600*24)
x -= d*(3600*24)

h = x//3600
x -= h*3600

m = x//60
x -= m*60

s = x

print(d, 'dia(s)')
print(h, 'hora(s)')
print(m, 'minuto(s)')
print(s, 'segundo(s)')

